src/components/ConnectProvider.svelte

```diff
@@ -0,0 +1,23 @@
+<script lang="ts">
+  export let close = () => {};
+  import Modal from '../components/Modal.svelte';
+  import DropDownMenu from './DropDownMenu.svelte';
+</script>
+
+<Modal {close}>
+  <div class="flex flex-col gap-7">
+    <h2 class="section-title-underlined">Connect Provider</h2>
+    <!-- Select a network dropdown goes here -->
```

Comment:
@rlaferla TODO

Reply:
TODOs to be resolved in https://github.com/LibertyDSNP/provider-dashboard/issues/113